### PR TITLE
Style selection layout renders data dynamicaly and new carousel

### PR DESCRIPTION
**Pull request by Doodlis**
new carousel and selection tab

### CODE REVIEW

File: src/components/OverviewComponent/Overview.js [hunk 1]
```diff
@@ -64,127 +58,142 @@ const Overview = (props) => {
     dispatch(fetchProductStyle(props.productId));
   }, []);
 
+
+  // expands image when expand icon is clicked on a top right corner of a picture
+  const toggleCrap = () => {
```
Comment (davidlichter) [inline src/components/OverviewComponent/Overview.js:63]: Might want to consider a different name for this function

**Review by davidlichter — DISMISSED** [review]
Looks good, but there are conflicts that need to be resolved in package.json, App.css, index.css, and yarn.lock.

File: src/components/OverviewComponent/Overview.js (hunk 1 above)
Comment (jessepye) [reply src/components/OverviewComponent/Overview.js:63]: lol

**Review by davidlichter — APPROVED** [review]
Updated

**Pull request merged**